cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -945,6 +960,7 @@ def update_nrpe_config(unused=None):
     current_unit = nrpe.get_nagios_unit_name()
     nrpe_setup = nrpe.NRPE(hostname=hostname)
     nrpe.add_init_service_checks(nrpe_setup, services, current_unit)
+    nrpe.add_init_service_checks(nrpe_setup, master_services, current_unit)
```

Comment:
why are we nrpe'ing both snap.*.daemon and *?

Reply:
I came to comment on this. Originally I was merging nrpe into the master_services array and ended up pulling it for time. I guess I didn’t pull it all, but this should be nuked.

Reply:
Ah, I missed this. Pulled it. Thanks!